config/manifests/bases/compliance-operator.clusterserviceversion.yaml

```diff
@@ -1557,4 +1557,5 @@ spec:
   provider:
     name: Red Hat Inc.
     url: www.redhat.com
+  replaces: 0.1.53
```

Comment:
Note for reviewers.

This doesn't really matter since the `config/` manifests are just used as templates for the `bundle` manifests that are generated from operator-sdk. And even though we're setting this here to an older version, we're going to update it in the `git-release` target since neither kustomize or operator-sdk supports setting this through the CLI.

Reply:
Should it be compliance-operator.v0.1.53? I am sorry, I am not quite sure. 
I tried to update it to compliance-operator.v0.1.53 in my local env, the index(which contains compliance-operator.v0.1.53) still failed to add the bundle image in this PR to the index. 

```
ERRO[0005] permissive mode disabled                      bundles="[quay.io/xiyuan/compliance-operator-bundle:pr-475]" error="Invalid bundle compliance-operator.v1.2.0, replaces nonexistent bundle 0.1.53"
Error: Invalid bundle compliance-operator.v1.2.0, replaces nonexistent bundle 0.1.53
```

Reply:
Interesting - it could be that it's failing because there isn't an existing, discoverable build for version 0.1.53. I can update and try with 0.1.61, which is the last release from the 0.1 release channel.

Reply:
Made a minor change to this in the latest update.